app/lib/import_export.rb

```diff
@@ -525,9 +525,6 @@ def get_export_data(patients, data)
 
       # Replace symptoms field with actual symptom fields
       if data[:assessments][:checked].include?(:symptoms)
-        data[:assessments][:headers] = data[:assessments][:checked].map { |field| ASSESSMENT_FIELD_NAMES[field] } if data[:assessments][:headers].nil?
```

Comment:
Please note: In the `update_assessment_headers` method I wrote that contains this line, I changed it to be _before_ checking if symptoms are checked rather than after (as it is here). I did this for two main reasons: 1) It doesn't make much sense that you would only populate column names if one of the columns was checked, 2) I don't actually think this line is ever hit and it's more of a fail safe. 

Let me know if anyone has any issues with this. @timwongj wrote this originally so his reasoning would probably be best, so if we want I can undo this change for this PR until we feel it's safer.